logging-infrastructure/README.md

```diff
@@ -0,0 +1,62 @@
+# Logging infrastructure
+
+As a Giant Swarm engineer, I want to be able to access a history of logs for components managing our platform in order to both investigate ongoing operational issues and provide details for incident reports.
+
+## Context
+
+Atlas started working on a logging infrastructure implementation using [Loki](https://grafana.com/docs/loki/latest/).
+So far we worked on a POC towards having a centralized setup, where a central Loki instance hosted by GiantSwarm ingests logs from our installations.
+There were some recent discussions where we learned about other requirements leaning towards a distributed setup, here are the pro and cons details of those discussions.
+
+### Distributed setup
+
+What we call distributed setup, is when there is 1 Loki instance hosted on each installation.
+So logs are stored and accessible on a per installation basis.
+
+#### Pros
+
+- Satisfy customer requirement towards data isolation (i.e. legal, or data privacy requirements)
+- More resilient setup towards failure
+- Keep our Platform as a Product model (self-sufficient installation can keep running alone)
+
+#### Cons
+
+- Heterogeneous Object storage (different provider, different requirements, different limitations)
+
```

Comment:
Another con is that we lose the use case of checking logs accross all installations which would be useful for things like the app platform crossplanes and so on (e.g. error trends accross installations/regions/providers to detect issues earlier than if we check at the installation level)

Reply:
cc @piontec as this is something you would have loved to have

Reply:
I kind of already added those point under centralized setup pros:

- `Global view (aka single pane of glass, cross-installation queries)`
- `Anomalies detections (multi installation correlated data)`

Reply:
Sure thing but the comments bring more context, or at least I think it does :D

Reply:
I added part of your comment in a later section as
`Anomalies detections (i.e. detect issues earlier via error/trends measures accross installations/regions/providers)`